Documentation/operations/upgrade.rst

```diff
@@ -322,6 +322,8 @@ communicating via the proxy must reconnect to re-establish connections.
 * The Cilium cluster name validation cannot be bypassed anymore, both for the local and
   remote clusters. The cluster name is strictly enforced to consist of at most 32 lower
   case alphanumeric characters and '-', start and end with an alphanumeric character.
+* The option to run a synchronous probe using ``cilium-health status --probe`` is no longer supported,
+  and now returns the results of the most recent cached probe.
```

Comment:
```suggestion
  it is now a hidden option that returns the results of the most recent cached probe.
```

Is there any plan to remove the option in a future version? If so, please mention it in the note.

Reply:
Thanks for the suggestion!